_code-samples/batch/js/singleAccountBatch.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Single Account Batch Transaction Example
+ *
+ * This example demonstrates how to use the Batch transactions feature (XLS-56)
+ * to create a single-account batch transaction that sends payments
+ * to multiple destinations in one atomic operation.
+ * Concept doc: https://xrpl.org/docs/concepts/transactions/batch-transactions
+ * Reference doc: https://xrpl.org/docs/references/protocol/transactions/types/batch
+*/
+
+import xrpl from "xrpl"
+
+const client = new xrpl.Client("wss://s.devnet.rippletest.net:51233/")
+await client.connect()
+
+// Create and fund wallets
+console.log("=== Funding new wallets from faucet... ===")
+const { wallet: sender } = await client.fundWallet()
+const { wallet: wallet1 } = await client.fundWallet()
+const { wallet: wallet2 } = await client.fundWallet()
+
+console.log(`Sender: ${sender.address}, Balance: ${await client.getXrpBalance(sender.address)} XRP`)
+console.log(`Wallet1: ${wallet1.address}, Balance: ${await client.getXrpBalance(wallet1.address)} XRP`)
+console.log(`Wallet2: ${wallet2.address}, Balance: ${await client.getXrpBalance(wallet2.address)} XRP`)
+
+// Create inner transactions  --------------------------------------------
+// REQUIRED: Inner transactions MUST have the tfInnerBatchTxn flag (0x40000000).
+// This marks them as part of a batch (allows Fee: 0 and empty SigningPubKey).
+
+// Transaction 1
+const payment1 = {
+  TransactionType: "Payment",
+  Account: sender.address,
+  Destination: wallet1.address,
+  Amount: xrpl.xrpToDrops(2),
+  Flags: xrpl.GlobalFlags.tfInnerBatchTxn // THIS IS REQUIRED
+}
```

Comment:
So in the `Batch` transaction page it says you have to set the `Fee` to 0 and have an empty `SigningPubKey`. Neither are defined here, but when I run the script it completes successfully. Maybe update that transaction page to say those fields can also be ommitted?